Description: a2dpsink should have a rank < GST_RANK_MARGINAL
Author: Sebastian Dröge <slomo@circular-chaos.org>
Origin: Sebastian Dröge <slomo@circular-chaos.org>
Updated by Nobuhiro Iwamatsu <iwamatsu@debian.org>
Debian BTS: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=571595

diff --git a/audio/gsta2dpsink.c b/audio/gsta2dpsink.c
index 930d14e..b7ec67c 100644
--- a/audio/gsta2dpsink.c
+++ b/audio/gsta2dpsink.c
@@ -725,6 +725,6 @@ static void gst_a2dp_sink_init(GstA2dpSink *self,
 gboolean gst_a2dp_sink_plugin_init(GstPlugin *plugin)
 {
 	return gst_element_register(plugin, "a2dpsink",
-			GST_RANK_MARGINAL, GST_TYPE_A2DP_SINK);
+			GST_RANK_MARGINAL-1, GST_TYPE_A2DP_SINK);
 }
 
-- 
1.7.5.4

